domain/dcsa_ebl_domain.yaml

```diff
@@ -343,16 +362,7 @@ components:
             displayedAddress:
               type: array
               items:
-                type: object
-                allOf:
-                  - type: object
-                    properties:
-                      addressLine:
-                        $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/addressLine'
-                  - type: object
-                    properties:
-                      addressLineNumber:
-                        $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/addressLineNumber'
+                $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/addressLine'
```

Comment:
I guess it is correct, that displayedAddress is only a single line insead of an entire address. Never the less wanted to rase this comment in order to discuss

Reply:
Yes - addressLine is just one line. It is a freetext field for Shippers to use. BackEnd we have an addressLineNumber - but this is not needed frontEnd as the order of the array is maintained